jenkins/vm-reset

```diff
@@ -61,18 +61,31 @@ validate_args() {
 
 wait_till() {
     local vm_id=$1
-    local state=$2
+    local interval=$2
+    local state=$3
+    local nr_steps
+
+    if [ $interval -eq 5 ]; then
```

Comment:
- How interval value changes?
- pass WAIT_TIMEOUT as an argument to `wait_till()` function and have just one way to calculate `nr_steps` based on `WAIT_TIMEOUT`.  See if interval can be kept constant.